mocked-api/handlers.ts

```diff
@@ -233,6 +233,21 @@ export const handlers = [
       );
     }
 
+    if (name.includes("quota-error")) {
+      return res(
+        ctx.status(402),
+        ctx.json({
+          kind: "ErrorsResponse",
+          items: [
+            {
+              ...error_quota_exceeded,
+              reason: `Creation of bridge was no successful due to exceeded quota'`,
```

Comment:
You can use the same reason message the API returns. You can find it inside the `errors.ts` you updated, it's in the comment before the new error code.

Given it's a generic message for both SE instances and processors, you can also move it inside `error_quota_exceeded`.